pkg/model/components/etcdmanager/model.go

```diff
@@ -68,25 +65,29 @@ func (b *EtcdManagerBuilder) Build(c *fi.ModelBuilderContext) error {
 			return fmt.Errorf("backupStore must be set for use with etcd-manager")
 		}
 
-		manifest, err := b.buildManifest(etcdCluster)
-		if err != nil {
-			return err
-		}
+		for _, member := range etcdCluster.Members {
+			instanceGroupName := fi.StringValue(member.InstanceGroup)
```

Comment:
It _might_ be that we should use the member name here.  On the one hand, it's the primary key and therefore maybe "more correct", and maybe avoids the problem of then wanting to lock instance groups to members.  On the other hand, it means users must be more aware of etcd member names, when they are probably more familiar with instance group names.

Reply:
Edit: but now I see that it the mapping in nodeup below becomes much more straightforward if we stick to instance groups...